--- a/jni/src/Android.mk
+++ b/jni/src/Android.mk
@@ -10,7 +10,7 @@ LOCAL_C_INCLUDES := $(LOCAL_PATH)/$(SDL_PATH)/include
 
 # Add your application source files here...
 LOCAL_SRC_FILES := $(SDL_PATH)/src/main/android/SDL_android_main.c \
-	YourSourceHere.c
+	mruby.c game.c
 
 LOCAL_SHARED_LIBRARIES := SDL2
 
--- a/jni/src/Android_static.mk
+++ b/jni/src/Android_static.mk
@@ -4,7 +4,7 @@ include $(CLEAR_VARS)
 
 LOCAL_MODULE := main
 
-LOCAL_SRC_FILES := YourSourceHere.c
+LOCAL_SRC_FILES := mruby.c game.c
 
 LOCAL_STATIC_LIBRARIES := SDL2_static

--- a/jni/mruby/Android.mk
+++ b/jni/mruby/Android.mk
@@ -3,7 +3,7 @@ LOCAL_PATH := $(call my-dir)
 include $(CLEAR_VARS)
 MRUBY_PATH = ../../../mruby
 LOCAL_MODULE    := mruby-prebuilt
-LOCAL_SRC_FILES := $(MRUBY_PATH)/build/androideabi/lib/libmruby.a
+LOCAL_SRC_FILES := $(MRUBY_PATH)/build/android-armeabi/lib/libmruby.a
 LOCAL_EXPORT_C_INCLUDES := $(LOCAL_PATH)/$(MRUBY_PATH)/include
 LOCAL_STATIC_LIBRARIES += android_support
 
